corehq/apps/app_manager/tests/test_suite.py

```diff
@@ -317,6 +317,83 @@ def test_case_tile_suite(self):
         self._test_generic_suite("app_case_tiles", "suite-case-tiles")
 
 
+class AdvancedModuleAsChildTest(SimpleTestCase, TestFileMixin):
+    "TODO - Add Case-dependency tests"
```

Comment:
As discussed, planning to support this and add the tests in next iteration of the feature.